src/NzbDrone.Core.Test/DecisionEngineTests/PrioritizeDownloadDecisionFixture.cs

```diff
@@ -183,6 +183,22 @@ public void should_put_torrent_above_usenet_when_torrent_is_preferred()
             qualifiedReports.First().RemoteAlbum.Release.DownloadProtocol.Should().Be(DownloadProtocol.Torrent);
         }
 
+        [Test]
+        public void should_prefer_discography_pack_above_single_album()
```

Comment:
What about when qualities differ? Do we want this to still be the case if only one or a few albums are monitored (specifically from something like album lists)?

Reply:
Currently the logic will prefer a higher quality or a quality revision over a discography. I should likely add a test for that also though. 

I need to add a test for the monitoring, I believe currently it would still download the discography which is likely not what we want. I'll modify the discography decision engine specification to prevent that.